ostree_upload_server/importers/base.py

```diff
@@ -83,3 +83,7 @@ def _apply_commit_to_repo(self, commit, ref):
         except:
             target_repo.abort_transaction(None)
             raise
+
+        logging.info("updating summary...")
```

Comment:
`update_repo_metadata` already has a log message that says `Updating repository metadata`. This seems redundant.